src/Ball.js

```diff
@@ -30,6 +31,8 @@ export class Ball {
     this.hasCollision = true
     this.survivor = false
     this.hasAppInstalled = hasAppInstalled
+    // a person can arbitrarily choose whether to respect auto-isolation or not
+    this.willRespectAutoIsolation = sketch.random(100) <= peopleRespectingAutoIsolationPercentage
```

Comment:
Are we planning to represent these?

Reply:
I think that can be useful to understand the importance to respect auto-isolation. What do you think about it?